config.yml

```diff
@@ -45,6 +45,10 @@ market:
     rate_update_time: 1h
     api: https://data.fixer.io/api
     api_key:
+  compound:
+    rate_update_time: 15m
+    quote_update_time: 15m
```

Comment:
can we use generic timeout? 

`market.rate_update_time `
`quote.quote_update_time `

this might be reused quite a lot by other providers

Reply:
Let's always use standard timeout, unless there is a need for custom